graph/docu/docu_04graph_03.md

```diff
@@ -0,0 +1 @@
+![](https://raw.githubusercontent.com/dkorolev/Bricks/master/graph/golden/love-Linux.png)
```

Comment:
Yes, it's to have the `README.md` file readable during the review. Good to fix and merge?